composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/TextFields.kt

```diff
@@ -143,7 +155,7 @@ object TextFields {
 
         Box(modifier, contentAlignment = Alignment.CenterEnd) {
             EditText(
-                errorMessage = if (!isValid) stringResource(R.string.password_is_required) else null,
+                errorMessage = if (! isValid) stringResource(R.string.password_is_required) else null,
```

Comment:
Why are these spaces added?

Reply:
Probably the android studio code clean-up

Reply:
I fixed corrected it